### ISSUE TITLE
Can't add integration - " Config flow could not be loaded: undefined "

### ISSUE THREAD

**Issue by flcmakerspace** [briis/smartweatherudp#43]
Disclaimer: I am new to Home Assistant, but I understand all the set-up, etc. 

When I attempt to add the Weatherflow integration, I'm getting the pictured error. If it doesn't give that error, then it just sits on the "please wait WeatherFlow- Local is being setup" screen. Any help would be appreciated.
![Screenshot from 2022-02-24 09-49-33](https://user-images.githubusercontent.com/36424943/155579856-ce3ace9f-f8f6-41b8-a069-a176cad08aab.png)

**Comment by natekspencer** [comment]
What is your HA setup (hardware, OS/Container/Supervised/Core and version)? And what version of this integration is installed? Typically what has to happen when adding an integration for the first time is that it downloads the required python modules. If there is an issue downloading one of those, it can hang or cause an error. Also, is there any messages in your logs (Supervisor/Core/etc)?

**Comment by flcmakerspace** [comment]
@natekspencer Thanks for reaching out.
Since my post, I actually have removed everything and installed HACS again. I found exactly what I was looking for https://github.com/briis/hass-weatherflow instead of the smartweatheredup, but I am still getting the same issue when trying to install the weatherflow.

This is my HA setup:
Version | core-2022.2.9
-- | --
Installation Type | Home Assistant OS
Development | false
Supervisor | true
Docker | true
User | root
Virtual Environment | false
Python Version | 3.9.7
Operating System Family | Linux
Operating System Version | 5.10.98
CPU Architecture | x86_64
Timezone | America/Los_Angeles

For https://github.com/briis/hass-weatherflow, I am using V1.0.4.
For HACS, I have 1.23.0

This is my supervisor log:
22-02-24 20:22:38 INFO (MainThread) [__main__] Initializing Supervisor setup
22-02-24 20:22:38 INFO (MainThread) [supervisor.bootstrap] Initializing Supervisor Sentry
22-02-24 12:22:38 INFO (MainThread) [supervisor.bootstrap] Seting up coresys for machine: qemux86-64
22-02-24 12:22:38 INFO (SyncWorker_0) [supervisor.docker.supervisor] Attaching to Supervisor homeassistant/amd64-hassio-supervisor with version 2022.01.1
22-02-24 12:22:38 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
22-02-24 12:22:38 WARNING (SyncWorker_1) [supervisor.docker.interface] Can't find homeassistant/amd64-hassio-supervisor for cleanup
22-02-24 12:22:38 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.INITIALIZE
22-02-24 12:22:38 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-02-24 12:22:38 INFO (MainThread) [__main__] Setting up Supervisor
22-02-24 12:22:38 INFO (MainThread) [supervisor.api] Starting API on 172.30.32.2
22-02-24 12:22:38 INFO (MainThread) [supervisor.hardware.monitor] Started Supervisor hardware monitor
22-02-24 12:22:38 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface io.hass.os
22-02-24 12:22:38 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.systemd1
22-02-24 12:22:38 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.login1
22-02-24 12:22:38 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.hostname1
22-02-24 12:22:38 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.timedate1
22-02-24 12:22:39 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface org.freedesktop.NetworkManager
22-02-24 12:22:39 INFO (MainThread) [supervisor.dbus.manager] Load dbus interface de.pengutronix.rauc
22-02-24 12:22:39 INFO (MainThread) [supervisor.host.info] Updating local host information
22-02-24 12:22:39 INFO (MainThread) [supervisor.host.services] Updating service information
22-02-24 12:22:39 INFO (MainThread) [supervisor.host.network] Updating local network information
22-02-24 12:22:39 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
22-02-24 12:22:39 INFO (MainThread) [supervisor.host.manager] Host information reload completed
22-02-24 12:22:39 INFO (MainThread) [supervisor.host.apparmor] Loading AppArmor Profiles: {'hassio-supervisor'}
22-02-24 12:22:39 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/amd64-hassio-cli with version 2021.12.0
22-02-24 12:22:39 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/amd64-hassio-dns with version 2021.06.0
22-02-24 12:22:39 INFO (MainThread) [supervisor.plugins.dns] Updated /etc/resolv.conf
22-02-24 12:22:39 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/amd64-hassio-audio with version 2021.07.0
22-02-24 12:22:39 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/amd64-hassio-observer with version 2021.10.0
22-02-24 12:22:39 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/amd64-hassio-multicast with version 2021.04.0
22-02-24 12:22:39 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
22-02-24 12:22:39 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/qemux86-64-homeassistant with version 2022.2.9
22-02-24 12:22:39 INFO (MainThread) [supervisor.homeassistant.secrets] Loaded 1 Home Assistant secrets
22-02-24 12:22:39 INFO (MainThread) [supervisor.os.manager] Detect Home Assistant Operating System 7.4 / BootSlot A
22-02-24 12:22:39 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/a0d7b954 repository
22-02-24 12:22:39 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/core repository
22-02-24 12:22:39 INFO (MainThread) [supervisor.store] Loading add-ons from store: 62 all - 62 new - 0 remove
22-02-24 12:22:39 INFO (MainThread) [supervisor.addons] Found 3 installed add-ons
22-02-24 12:22:39 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to homeassistant/amd64-addon-ssh with version 9.3.0
22-02-24 12:22:39 INFO (SyncWorker_2) [supervisor.docker.interface] Attaching to ghcr.io/hassio-addons/jupyterlab/amd64 with version 0.6.0
22-02-24 12:22:39 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to homeassistant/amd64-addon-git_pull with version 7.13.1
22-02-24 12:22:39 INFO (MainThread) [supervisor.backups.manager] Found 0 backup files
22-02-24 12:22:39 INFO (MainThread) [supervisor.discovery] Loaded 0 messages
22-02-24 12:22:39 INFO (MainThread) [supervisor.ingress] Loaded 6 ingress sessions
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.SETUP
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.check] System checks complete
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.SETUP
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-02-24 12:22:39 INFO (MainThread) [supervisor.jobs] 'ResolutionFixup.run_autofix' blocked from execution, system is not running - CoreState.SETUP
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.SETUP
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-02-24 12:22:39 INFO (MainThread) [__main__] Running Supervisor
22-02-24 12:22:39 INFO (MainThread) [supervisor.os.manager] Rauc: A - marked slot kernel.0 as good
22-02-24 12:22:39 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.INITIALIZE' starting 0 add-ons
22-02-24 12:22:39 INFO (MainThread) [supervisor.core] Supervisor reboot detected
22-02-24 12:22:39 INFO (MainThread) [supervisor.misc.tasks] All core tasks are scheduled
22-02-24 12:22:39 INFO (MainThread) [supervisor.core] Supervisor is up and running
22-02-24 12:22:39 INFO (MainThread) [supervisor.host.info] Updating local host information
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.module] Create new suggestion SuggestionType.CREATE_FULL_BACKUP - ContextType.SYSTEM / None
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.PLUGIN
22-02-24 12:22:39 INFO (MainThread) [supervisor.host.services] Updating service information
22-02-24 12:22:39 INFO (MainThread) [supervisor.host.network] Updating local network information
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.CORE
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.SUPERVISOR
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.check] System checks complete
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
22-02-24 12:22:39 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
22-02-24 12:22:40 INFO (MainThread) [supervisor.host.manager] Host information reload completed
22-02-24 12:24:53 WARNING (MainThread) [supervisor.homeassistant.websocket] Connection is closed
22-02-24 12:24:53 WARNING (MainThread) [supervisor.misc.tasks] Watchdog miss API response from Home Assistant
22-02-24 12:25:08 WARNING (MainThread) [supervisor.misc.tasks] Watchdog found a problem with Home Assistant Docker!
22-02-24 12:25:08 INFO (SyncWorker_1) [supervisor.docker.interface] Starting homeassistant
22-02-24 12:25:08 INFO (MainThread) [supervisor.homeassistant.core] Wait until Home Assistant is ready
22-02-24 12:25:11 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
22-02-24 12:25:11 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-02-24 12:25:18 INFO (MainThread) [supervisor.homeassistant.core] Detect a running Home Assistant instance
22-02-24 12:29:07 ERROR (MainThread) [supervisor.misc.tasks] Watchdog found a problem with Home Assistant API!
22-02-24 12:29:07 INFO (SyncWorker_2) [supervisor.docker.interface] Restarting ghcr.io/home-assistant/qemux86-64-homeassistant
22-02-24 12:29:11 INFO (MainThread) [supervisor.homeassistant.core] Wait until Home Assistant is ready
22-02-24 12:29:13 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
22-02-24 12:29:13 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-02-24 12:29:21 INFO (MainThread) [supervisor.homeassistant.core] Detect a running Home Assistant instance
22-02-24 12:35:28 WARNING (MainThread) [supervisor.misc.tasks] Watchdog found a problem with Home Assistant Docker!
22-02-24 12:35:28 INFO (SyncWorker_0) [supervisor.docker.interface] Starting homeassistant
22-02-24 12:35:28 INFO (MainThread) [supervisor.homeassistant.core] Wait until Home Assistant is ready
22-02-24 12:35:31 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
22-02-24 12:35:31 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
22-02-24 12:35:39 INFO (MainThread) [supervisor.homeassistant.core] Detect a running Home Assistant instance
22-02-24 12:42:12 INFO (MainThread) [supervisor.backups.manager] Found 0 backup files
22-02-24 12:42:12 INFO (MainThread) [supervisor.backups.manager] Found 0 backup files

My Core Log:
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
2022-02-24 12:16:51 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-02-24 12:17:13 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from flcinnovation.flc.ad.losrios.edu (10.50.129.246). (Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:94.0) Gecko/20100101 Firefox/94.0)
2022-02-24 12:19:11 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-02-24 12:21:34 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration weatherflow which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-02-24 12:21:34 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
2022-02-24 12:21:46 ERROR (MainThread) [homeassistant.components.hassio.handler] Client error on /homeassistant/restart request Server disconnected
[finish] process exit code 0
s6-svscanctl: fatal: unable to control /var/run/s6/services: supervisor not listening
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
2022-02-24 12:21:52 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-02-24 12:21:52 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration weatherflow which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
[finish] process exit code 256
[finish] process received signal 9
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
2022-02-24 12:25:10 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-02-24 12:25:10 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration weatherflow which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-02-24 12:25:10 WARNING (Recorder) [homeassistant.components.recorder.util] The system could not validate that the sqlite3 database at //config/home-assistant_v2.db was shutdown cleanly
2022-02-24 12:25:10 WARNING (Recorder) [homeassistant.components.recorder.util] Ended unfinished session (id=49 from 2022-02-24 20:21:52.321824)
[finish] process exit code 256
[finish] process received signal 9
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
2022-02-24 12:29:12 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-02-24 12:29:12 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration weatherflow which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-02-24 12:29:12 WARNING (Recorder) [homeassistant.components.recorder.util] The system could not validate that the sqlite3 database at //config/home-assistant_v2.db was shutdown cleanly
2022-02-24 12:29:12 WARNING (Recorder) [homeassistant.components.recorder.util] Ended unfinished session (id=50 from 2022-02-24 20:25:10.446558)
[finish] process exit code 256
[finish] process received signal 9
[cont-finish.d] executing container finish scripts...
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] done.
[services.d] starting services
[services.d] done.
2022-02-24 12:35:30 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-02-24 12:35:30 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration weatherflow which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-02-24 12:35:30 WARNING (Recorder) [homeassistant.components.recorder.util] The system could not validate that the sqlite3 database at //config/home-assistant_v2.db was shutdown cleanly
2022-02-24 12:35:30 WARNING (Recorder) [homeassistant.components.recorder.util] Ended unfinished session (id=51 from 2022-02-24 20:29:12.915890)

Thank you!
